x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -6,6 +6,11 @@
  */
 import { PublicMethodsOf } from '@kbn/utility-types';
 import { decodeVersion, encodeHitVersion } from '@kbn/securitysolution-es-utils';
+import {
+  mapConsumerToIndexName,
+  validFeatureIds,
+  isValidFeatureId,
+} from '@kbn/rule-data-utils/target/alerts_as_data_rbac';
```

Comment:
Is there a way to remove the target path and just import directly from `@kbn/rule-data-utils` similar to some of the other packages listed above? This is a nit, just think it's cleaner.

Reply:
Def! If it's ok to do as follow up to not add more to this PR, I'll follow up on this.